wagtail/admin/api/views.py

```diff
@@ -26,6 +28,23 @@ class PagesAdminAPIViewSet(PagesAPIViewSet):
     base_serializer_class = AdminPageSerializer
     authentication_classes = [SessionAuthentication]
 
+    @property
+    def renderer_classes(self):
```

Comment:
Can we put this in https://github.com/wagtail/wagtail/blob/bb8800927c74207fe7de9f4efca8a8961e2f95ab/wagtail/api/v2/views.py#L37-L38 instead as per https://github.com/wagtail/wagtail/issues/13265#issuecomment-3107821235?